services/legacy-service.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+const BaseService = require('./base')
+
+// registerFn: ({ camp, cache }) => { camp.route(/.../, cache(...)) }
+class LegacyService extends BaseService {
+  static registerLegacyRouteHandler({ camp, cache }) {
```

Comment:
By making this a dictionary, we can easily inject more collaborators from `server.js` via `register()`, if that turns out to be necessary.